### ISSUE TITLE
Rationale Dialog crashes app when used from android.app.Fragment

### ISSUE THREAD

**Issue by j-garin** [googlesamples/easypermissions#98]
Hello.
Looks like full android.app.Fragment support is still an issue. When I call easypermissions from android.app.Fragment and the user clicks on aby button in dialog, the  app crashes. Here are the logs:

```
                                     java.lang.ClassCastException: com.example.TestFragment cannot be cast to android.support.v4.app.FragmentActivity
                                         at pub.devrel.easypermissions.RationaleDialogClickListener.onClick(RationaleDialogClickListener.java:57)

```

**Comment by samtstern** [comment]
Looks like this would be fixed by #78, which has been merged into master but not released yet.  I will do a new release soon!  Sorry about the delay here.

**Comment by samtstern** [comment]
@j-garin you should be able to install a build from master if you're comfortable with that, happy to explain more if you need me to.

**Comment by samtstern** [comment]
@j-garin just released `0.3.1` with this fix
